src/utils/request.js

```diff
@@ -118,8 +118,9 @@ export default function(url, method, options, customErrorHandler) {
       bakedOptions = GET_OPTIONS;
   }
 
+  const requestUrl = url.slice(0, 4) === 'http' ? url : `${endpoint}/${url}`;
```

Comment:
this allows us to make cross domain requests

Reply:
Ah makes sense here...